### PR TITLE
[Fix #1773] Fix typo ('strptime' to 'strftime')

### LINKED ISSUE #1773: Wrong Time.zone suggestion
For this code
````
  def validate_datetime(datetime = nil, format = '%d/%m/%Y %H:%M:%S')
    return nil if datetime.nil? || datetime.empty?
    Time.strptime(datetime, format).in_time_zone
  rescue StandardError
    nil
  end
````
I get this message
````
app/controllers/application_controller.rb:48:10: C: Do not use Time.[:strptime] without zone. Use #Time.zone.[:strptime] instead.
    Time.strptime(datetime, format).in_time_zone

````

but Time.zone.strptime doesn't exist....

rubocop 0.30.0
rails 4.2.1

Regards

### PR DESCRIPTION
**Pull request by palkan**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
 ## 0.30.0 (06/04/2015)
 
 ### New features
+* [#1773](https://github.com/bbatsov/rubocop/pull/1773): Fix type ('strptime' -> 'strftime') in `Rails/TimeZone`. ([@palkan][])
```
Comment (bbatsov) [inline CHANGELOG.md]: This should be in the `Bugs fixed` section for `master`.

Comment (palkan) [inline CHANGELOG.md]: Done

File: CHANGELOG.md [hunk 2]
```diff
@@ -42,6 +42,7 @@
 
 ### Bugs fixed
 
+* [#1773](https://github.com/bbatsov/rubocop/pull/1773): Fix type ('strptime' -> 'strftime') in `Rails/TimeZone`. ([@palkan][])
```
Comment (bbatsov) [inline CHANGELOG.md]: This is the wrong section again. :-) Update your master branch.

Comment (palkan) [inline CHANGELOG.md]: Oh, sorry) Fixed again.

**Comment by bbatsov** [comment]
:+1:

**Pull request merged**